packages/@aws-cdk/aws-servicecatalogappregistry/lib/private/util.ts

```diff
@@ -0,0 +1,10 @@
+import * as crypto from 'crypto';
+
+/**
+ * Generates a unique hash identfifer using SHA256 encryption algorithm
+ */
+export function hashValues(...ids: string[]): string {
```

Comment:
Here, I would probably call the parameter `values` - they don't have to be IDs for the hashing to work.